csi/moac/test/csi_test.js

```diff
@@ -392,7 +424,7 @@ module.exports = function() {
               block: {},
             },
           ],
-          parameters: { repl: '3' },
+          parameters: { repl: '3', protocol: 'nexus_nbd' },
```

Comment:
it does not matter too much as the createVolume method just passes the string from input to output value, though it might be better to pass a correct value for the parameter: `nbd`. The same comment applies to line 453.

Reply:
Done. https://github.com/openebs/Mayastor/pull/150/commits/2c509c7bc5230fda308031b496a234ad631b66ad